src/themes/theme.js

```diff
@@ -0,0 +1,41 @@
+import tw from "twin.macro";
+
+const greyAndGrayBorder = tw`border-gray-300 text-gray-700 bg-white-100 hover:bg-gray-50`;
```

Comment:
Please rename to `greyBorder` and apply below.  Thanks.

Reply:
fixed